search-enterprise/latest/install/index.md

```diff
@@ -0,0 +1,405 @@
+* **Content:** indexes content.
+* **Path:** indexes the path of a node.
+
+Additionally, a Kubernetes job will be started to reindex existing content in Search Enterprise. It is recommended you only run this job at the initial startup. You can enable or disable the setting in the `alfresco-elasticsearch-connector.reindexing.enabled` property file by using `true` or `false`.
+
+To deploy Content Services with the Elasticsearch connector:
+
+ ```bash
+ helm install acs alfresco/alfresco-content-services \
+ --values esc_values.yaml \
+ --set externalPort="80" \
+ --set externalProtocol="http" \
+ --set externalHost="localhost" \
+ --set global.alfrescoRegistryPullSecrets=my-registry-secrets \
+ --set repository.replicaCount=1 \
+ --set transformrouter.replicaCount=1 \
+ --set pdfrenderer.replicaCount=1 \
+ --set imagemagick.replicaCount=1 \
+ --set libreoffice.replicaCount=1 \
+ --set tika.replicaCount=1 \
+ --set transformmisc.replicaCount=1 \
+ --set postgresql-syncservice.resources.requests.memory="500Mi" \
+ --set postgresql-syncservice.resources.limits.memory="500Mi" \
+ --set postgresql.resources.requests.memory="500Mi" \
+ --set postgresql.resources.limits.memory="500Mi" \
+ --set alfresco-search.resources.requests.memory="1000Mi" \
+ --set alfresco-search.resources.limits.memory="1000Mi" \
+ --set share.resources.limits.memory="1500Mi" \
+ --set share.resources.requests.memory="1500Mi" \
+ --set repository.resources.limits.memory="2500Mi" \
+ --set repository.resources.requests.memory="2500Mi"\
+ --timeout 10m0s \
+ --namespace=alfresco
+ ```
+
+If you are using Docker Desktop locally, you must set `antiAffinity` to `soft` and it is recommended you reduce the Elasticsearch server resources:
+
+```docker
+ elasticsearch:
+   enabled: true
+   antiAffinity: "soft"
+
+   # Shrink default JVM heap.
+   esJavaOpts: "-Xmx128m -Xms128m"
+
+   # Allocate smaller chunks of memory per pod.
+   resources:
+     requests:
+       cpu: "100m"
+       memory: "512M"
+     limits:
+       cpu: "1000m"
+       memory: "512M"
+
+   # Request smaller persistent volumes.
+   volumeClaimTemplate:
+     accessModes: [ "ReadWriteOnce" ]
+     storageClassName: "hostpath"
+     resources:
+       requests:
+         storage: 100M
+```
+
+When the system is up and running, you can access the Kibana console using port forwarding:
+
+```bash
+ kubectl port-forward service/acs-kibana 5601:5601 -n alfresco
+```
+
+and then you can access the console http://localhost:5601/app/kibana.
+
+If you need access to the Elasticsearch server directly you have to perform the same operation:
+
+```bash
+ kubectl port-forward service/elasticsearch-master 9200:9200 -n alfresco
+```
+
+and then you can access the server http://localhost:9200/.
+
+More properties that can be used to configure the chart are available [here](https://github.com/Alfresco/acs-deployment/tree/master/helm/alfresco-content-services/charts/alfresco-elasticsearch-connector/README.md){:target="_blank"}.
```

Comment:
Link goes to page not found

Reply:
This page will be available on release day, cheers.